lib/saml_idp_constants.rb

```diff
@@ -6,12 +6,22 @@ module Constants
       LOA3_AUTHN_CONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/3'.freeze
       IAL1_AUTHN_CONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/ial/1'.freeze
       IAL2_AUTHN_CONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/ial/2'.freeze
+      IAL3_AUTHN_CONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/ial/3'.freeze
```

Comment:
I understand that we want something "more" than ial2, but I feel fairly strong that calling it `ial3` is a misnomer

From the the [NIST 800-63-3](https://pages.nist.gov/800-63-3/sp800-63-3.html) docs:

> IAL3: Physical presence is required for identity proofing. Identifying attributes must be verified by an authorized and trained representative of the CSP. As with IAL2, attributes can be asserted by CSPs to RPs in support of pseudonymous identity with verified attributes.

What if we added something like `/ial/2?strict=true`? We could convert the int column in the DB to a varchar and store things like `2.strict` if needed

Reply:
Yeah I can change it.  It's really just an enum